Orders/Order.java

```diff
@@ -0,0 +1,117 @@
+package Orders;
+
+import java.util.ArrayList;
+import java.util.Hashtable;
+
+public class Order
+{
+    private Integer idOrder;
+    private Integer idUser;
+    private Integer idRestaurant;
+    private Double value;
+    private String paymentMethod;
+    private Hashtable<Product, Integer> productList = new Hashtable<Product, Integer>();

```

Comment:
what was the reason behind using Hashtable and not a Map?

Reply:
Also, please use the interface name on the left part, and diamond operator on the right side, without the specific types since they will be inferred from left